pkg/cidata/cidata.TEMPLATE.d/boot/40-install-containerd.sh

```diff
@@ -8,18 +8,22 @@ fi
 # This script does not work unless systemd is available
 command -v systemctl >/dev/null 2>&1 || exit 0
 
-# Extract bin/nerdctl and compare whether it is newer than the current /usr/local/bin/nerdctl (if already exists).
-# Takes 4-5 seconds. (FIXME: optimize)
-tmp_extract_nerdctl="$(mktemp -d)"
-tar Cxzf "${tmp_extract_nerdctl}" "${LIMA_CIDATA_MNT}"/nerdctl-full.tgz bin/nerdctl
-
-if [ ! -f "${LIMA_CIDATA_GUEST_INSTALL_PREFIX}"/bin/nerdctl ] || [[ "${tmp_extract_nerdctl}"/bin/nerdctl -nt "${LIMA_CIDATA_GUEST_INSTALL_PREFIX}"/bin/nerdctl ]]; then
+# Check nerdctl version and compare whether it is different than the current /usr/local/bin/nerdctl (if already exists).
+if [ ! -f "${LIMA_CIDATA_MNT}"/nerdctl-version.txt ]; then
+	tmp_extract_nerdctl="$(mktemp -d)"
+	tar Cxzf "${tmp_extract_nerdctl}" "${LIMA_CIDATA_MNT}"/nerdctl-full.tgz bin/nerdctl
+	tmp_version_nerdctl="$("${tmp_extract_nerdctl}"/bin/nerdctl --version)"
+	rm -rf "${tmp_extract_nerdctl}"
+else
+	tmp_version_nerdctl="$(cat "${LIMA_CIDATA_MNT}"/nerdctl-version.txt)"
+fi
+if [ ! -f "${LIMA_CIDATA_GUEST_INSTALL_PREFIX}"/bin/nerdctl ] || [ "$tmp_version_nerdctl" != "$("${LIMA_CIDATA_GUEST_INSTALL_PREFIX}"/bin/nerdctl --version)" ]; then
```

Comment:
The user may install a custom version of nerdctl by themselves inside `limactl shell`, so we have to continue comparing the timestamp

Reply:
But if they can update the nerdctl binary _without_ changing the version, then I don't think this PR would work...

Reply:
It was assuming that the nerdctl version would be something like `0.18.0-74-g5e3dc32`, for custom versions